projects/charger/inc/generic_can.h

```diff
@@ -0,0 +1,49 @@
+#pragma once
+// Module for abstracting CAN implementation details.
```

Comment:
It would be nice to have some basic usage examples since we don't use this paradigm anywhere else

Reply:
Done.